docs/en/guides/improving-query-performance/sparse-primary-indexes/sparse-primary-indexes-design.md

```diff
@@ -229,9 +234,94 @@ In total the index has 1083 entries for our table with 8.87 million rows and 108
 <img src={require('./images/sparse-primary-indexes-03b.png').default} class="image"/>
 
 :::note
-The primary index file is completely loaded into the main memory. If the file is larger than the available free memory space then ClickHouse will raise an error.
+- The primary index file is completely loaded into the main memory. If the file is larger than the available free memory space then ClickHouse will raise an error.
 :::
 
+<details>
+    <summary><font color="black">
+    Inspecting the content of the primary index
+    </font></summary>
+    <p><font color="black">
+
+On a self-managed ClickHouse cluster we can use the <a href="https://clickhouse.com/docs/en/sql-reference/table-functions/file/" target="_blank"><font color="blue">file table function</font></a> for inspecting the content of the primary index of our example table.
+
+For that we first need to copy the primary index file into the <a href="https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings/#server_configuration_parameters-user_files_path" target="_blank"><font color="blue">user_files_path</font></a> of a node from the running cluster:
+<ul>
+<li>Step 1: Get part-path that contains the primary index file</li>
```

Comment:
It looks like your editor generated HTML instead of Markdown

Reply:
Can you have it generate Markdown instead?

Reply:
Unfortunately, inside a `<details>` element (a nice by default folded "for more/advanced info read this" feature), Markdown doesn't work correctly:

```
<details>
    <summary>
    Inspecting the content of the primary index
    </summary>
- list item 1
- list item 2
</details>
```

will be rendered as

<img width="986" alt="Screenshot 2022-12-29 at 11 06 18" src="https://user-images.githubusercontent.com/97666923/209942722-069d5513-80a0-4d26-bc07-b141129a42fb.png">

Therefore my 'workaround' of using HTML instead.